src/components/Dialogs/StakingProvider.tsx

```diff
@@ -22,7 +23,13 @@ import {
 } from "react";
 import { StakingSource } from "./StakingDialog/StakingDialog";
 
-const supportedPools: StakingPool[] = [LINEAR_POOL, STNEAR_POOL];
+const getSupportedPools = (networkId: string): StakingPool[] => {
+  const base = [LINEAR_POOL, STNEAR_POOL];
+  if (networkId === "mainnet") {
+    return [...base, RNEAR_POOL];
+  }
```

Comment:
As mentioned in previous PR, I don't think we need to check for `mainnet` here as we don't support anything on `testnet` anymore